sdk/python/requirements/py3.10-requirements.txt

```diff
@@ -137,32 +139,34 @@ protobuf==4.23.3
     #   grpcio-tools
     #   mypy-protobuf
     #   proto-plus
-pyarrow==14.0.1
+pyarrow==15.0.0
     # via feast (setup.py)
-pydantic==1.10.13
+pydantic==2.6.1
```

Comment:
Is there a specific reason for requiring 2.6.1? Otherwise, would you consider using something like `2.0.0` to allow for better compatibility for downstream users of this package?

Reply:
I didn't specify a version in the setup.py, so it loads the latest one :p.   let me find a "safe" version around 2.0.0. @sudohainguyen Harry, what's your opinion?

Reply:
the txt files are just for CI workflows 😄 
installation from end users should be fine since we set the boundaries in setup.py

Reply:
Apologies, the excitement got the better of me. Thanks for the clear up 💪